src/content/guides/getting-started.md

```diff
@@ -25,6 +25,7 @@ First let's create a directory, initialize npm, [install webpack locally](/guide
 ``` bash
 mkdir webpack-demo && cd webpack-demo
 npm init -y
+npm install --save-dev webpack
```

Comment:
Remove this line. We already show the installation for `webpack` and `webpack-cli` in the next line.
```bash
npm install webpack webpack-cli --save-dev
```